scripts/wsl/auto-install.sh

```
@@ -0,0 +1,122 @@
+
+print_script_step "Applying configuration changes (WSL, no reboot required)"
+sudo sysctl -p || true
+sudo modprobe ip6table_filter 2>/dev/null || echo "ip6table_filter module not available in the WSL kernel"
+
+# The SDK image is published for arm64 only, so it must be built locally
+# before the test collections setup, which installs the sample apps from it.
+# If the build fails, follow the printed guidance and finish the setup with
+# scripts/wsl/build-local-sdk-image.sh and scripts/wsl/update.sh (see
+# README.md).
+print_script_step "Building the SDK image locally (takes about an hour)"
+$WSL_SCRIPT_DIR/build-local-sdk-image.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)